dipper: audio: Enabled IIR0/Anc Filters

Bhavukverma17 opened the pull request:
Fix for ANC not working in Russian Manufatured Dipper (Mi8) Units due to different IIR0/Anc Frequency.
This issue was causing Huge noise  reflecting during Phone-calls.
Test: Call Someone and hear the sounds in reciever, There should be no noise.

Bhavukverma17 committed:
IIR0 Anc fix for Dipper

jjpprrrr commented on audio/mixer_paths_overlay_static.xml:
```diff
@@ -22,6 +22,14 @@
     <ctl name="QUAT_MI2S_TX SampleRate" value="KHZ_48" />
     <ctl name="External AMIC2 Mux" value="Dual_ADC" />
+    
+<!--IIR0 Voice/ Anc-->
```
Please fix the indentation of the comment.

jjpprrrr reviewed (changes requested):
Can you please also update the commit message to follow the format of this repo?

1. Start the subject line with "dipper: audio: " because the change is under the audio/ directory.
2. Please explain the issue you are trying to fix in the body of the commit message.
3. Please also include a test method so that other people can verify the commit. For example, say "Test: call someone on speaker and there should not be any echo."

Bhavukverma17 committed:
dipper: audio: Enabled IIR0 Filters

Fix for ANC not working in Russian Manufatured Dipper (Mi8) Units due to different IIR0/Anc Frequency.
This issue was causing Huge noise  reflecting during Phone-calls.
Test: Call Someone and hear the sounds in reciever, There should be no noise.

jjpprrrr reviewed (changes requested):
Please squash the two commits into one. Thank you.
